apps/site/assets/ts/stop/__tests__/StopPageRedesignTest.tsx

```diff
@@ -85,28 +118,103 @@ describe("StopPageRedesign", () => {
     expect(mapPolylines).toHaveLength(10);
   });
 
+  const dateFormatter = (date: Date): string => {
+    return format(date, "yyyy-M-d HH:mm");
+  };
```

Comment:
suggestion (non-blocking): Perhaps this would be useful in a test helper with a more descriptive name like `alertDateFormatter`